intelmq/bots/experts/domain_suffix/expert.py

```diff
@@ -54,12 +54,12 @@ def process(self):
     @staticmethod
     def check(parameters):
         if not os.path.exists(parameters.get('suffix_file', '')):
-            return [["error", "File given as parameter 'suffix_file' does not exist."]]
+            return [['error', 'File given as parameter \'suffix_file\' does not exist.']]
```

Comment:
There is no advantage, just played around and thought about a new standard in intelmq using single quotes outside and double quotes inside f-strings. But in order to avoid confusion, my second guess was using single quotes everytime and use double quotes inside. But as I was unsure if I will break some tests, rely on regex, I just escaped them for now.

Reply:
https://github.com/kenshoo/python-style-guide#user-content-double-vs-single-quotes
says
> 
    Prefer double quotes when your string has multiple words or when formatting ("hi bob").
    If the string is a single word use single quotes ('bob').

https://peps.python.org/pep-0008/#string-quotes says

> In Python, single-quoted strings and double-quoted strings are the same. This PEP does not make a recommendation for this. Pick a rule and stick to it. When a string contains single or double quote characters, however, use the other one to avoid backslashes in the string. It improves readability.
> 
> For triple-quoted strings, always use double quote characters to be consistent with the docstring convention in [PEP 257](https://peps.python.org/pep-0257).

These are just two examples of multiple (and we are not obliged to stick to them), but I agree with them.

I appreciate your conversion to format-strings, this makes the code much cleaner, more readable and a bit more efficient. But I don't see a reason to change the quotes if we need to escape inner quotes.

Reply:
@sebix As mentioned in PEP8, i've removed all backslashes now.

Reply:
IntelMQ is now also PEP257 compliant.